packages/terra-data-grid/src/DataGrid.jsx

```diff
@@ -126,6 +126,11 @@ const propTypes = {
    * rendered to allow for row selection to occur.
    */
   hasSelectableRows: PropTypes.bool,
+
+  /**
+   * Boolean indicating whether or not the DataGrid should hide the column headers.
+   */
+  hasColumnHeaders: PropTypes.bool,
```

Comment:
It has column headers. They're just invisible 🫥
```suggestion
  hasVisibleColumnHeaders: PropTypes.bool,
```